core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/orchestration/BootstrapMachineHandler.java

```diff
@@ -8,47 +8,55 @@
 
 import com.sequenceiq.cloudbreak.common.event.Selectable;
 import com.sequenceiq.cloudbreak.core.bootstrap.service.ClusterBootstrapper;
-import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.cloudbreak.core.flow2.cluster.provision.service.ClusterProxyService;
 import com.sequenceiq.cloudbreak.reactor.api.event.orchestration.BootstrapMachinesFailed;
 import com.sequenceiq.cloudbreak.reactor.api.event.orchestration.BootstrapMachinesRequest;
 import com.sequenceiq.cloudbreak.reactor.api.event.orchestration.BootstrapMachinesSuccess;
-import com.sequenceiq.flow.reactor.api.handler.EventHandler;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
 
 import reactor.bus.Event;
-import reactor.bus.EventBus;
 
 @Component
-public class BootstrapMachineHandler implements EventHandler<BootstrapMachinesRequest> {
+public class BootstrapMachineHandler extends ExceptionCatcherEventHandler<BootstrapMachinesRequest> {
```

Comment:
oh-oh :)
I did this because the other handler is deprecated on our side and the new one has default error handling forced. Should I really create UT for this change? :)

Reply:
Discussed offline, this can be tackled in a separate PR.